libs/core/src/lib/dynamic-page/dynamic-page.component.scss

```diff
@@ -27,7 +27,7 @@
     height: 100%;
 
     .fd-bar--floating-footer {
-        z-index: 2;
+        z-index: 6;
```

Comment:
Looks like it's not related to timeline.

Reply:
Wrong branch, thanks!